server/routerlicious/packages/services-telemetry/src/lumberjackCommonTestUtils.ts

```diff
@@ -64,6 +64,7 @@ export class TestSensitiveException extends Error {
 	public cookie = "cookie";
 	public token = "token";
 	public secret = "secret";
+	public authorization = "authorization";
```

Comment:
I don't think the Redis error has this keyword.
We leaked JWT token here because Redis error includes input args and input args has JWT tokens, there is no authorization keyword in error log. One way to validate your change is to see if it worked for the old santizeError scenario. Or this sanitize error is not only based on keyword?

The error log is like this: {
    "command": {
        "name": "set",
        "args": [
            "page:<leaked JWT token string>"
        ]
    }
}

Reply:
These test cases are for the previous sanitization logic, which is for unknown cases. My bad, though, I will include test cases for the base sanitization formatter.

Reply:
Sorry, my bad. I didn't read carefully and looks like your logic already covered the Redis scenario in BaseSanitizationLumberFormatter and SanitizationLumberFormatter will inherit the logic from base class

Reply:
No worries! Thanks for pointing it out, though. I will include BaseSanitizationLumberFormatter test cases.